debian/changelog

```diff
@@ -1,3 +1,9 @@
+firefox (104.0) jammy; urgency=medium
+
+  * https://www.mozilla.org/en-US/firefox/104.0/releasenotes/
+
+ -- Jeremy Soller <jeremy@system76.com>  Fri, 26 Jul 2022 18:30:00 +0000
```

Comment:
Not sure we should be signing the changelog as Jeremy if he's not the one making the PR. Also, the date on this changelog entry is a day before the previous entry (and a month before now.)

Reply:
Hey! I have updated the date, let me know what you want me to do with the signing!

Reply:
Are you updating that changelog manually? You should probably be using [dch](https://manpages.debian.org/bullseye-backports/devscripts/dch.1.en.html) for that.

Reply:
Thanks! Regardless of how it's done, you're welcome to put your own name & email address in since you're the one who made the update.

Reply:
thanks! didn't knew about `dch` will use that from next time. Also, updated the name & email address.